### DISCUSSION TITLE
Windows/Function  + Arrow keys to "snap" windows to display/multiple displays

### DISCUSSION THREAD

**Discussion by LeoFurnari** [runelite/runelite#15330] [Q&A] [answered]
You'll see this in something like google chrome, where if you click "Windows + Right Arrow", the window will snap to the right side, subsequently to the left side of the adjacent monitor. This feature currently doesn't exist in runelite, only un-maximizing with "Windows + Down Arrow".

**Comment by Alexsuperfly** [comment, marked as answer]
Disable custom window chrome in "RuneLite" settings
